functional/exec_test.go

```diff
@@ -13,3 +13,104 @@
 // limitations under the License.
 
 package functional
+
+import (
+	"fmt"
+	"time"
+
+	. "github.com/clearcontainers/tests"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+)
+
+var sleepingContainerWorkload = []string{"sh", "-c", "sleep 1000"}
+
+func execDetachTiming(detach bool) TableEntry {
+	comparator := ">="
+	if detach {
+		comparator = "<"
+	}
+
+	timeout := 1
+
+	process := Process{
+		Workload: []string{"/bin/sleep", fmt.Sprintf("%d", timeout)},
+		Detach:   detach,
+	}
+
+	return Entry(fmt.Sprintf("check time as detach=%t", detach), process, comparator, timeout)
+}
+
+func execDetachOutput(detach bool) TableEntry {
+	outputHelloWorld := "HelloWorld"
+
+	process := Process{
+		Workload: []string{"echo", outputHelloWorld},
+		Detach:   detach,
+	}
+
+	expectedOutput := outputHelloWorld
+	if detach {
+		expectedOutput = ""
+	}
+
+	return Entry(fmt.Sprintf("check output as detach=%t", detach), process, expectedOutput)
+}
+
+var _ = Describe("exec", func() {
+	var (
+		container *Container
+		exitCode  int
+		err       error
+	)
+
+	BeforeEach(func() {
+		container, err = NewContainer(sleepingContainerWorkload, true)
+		Ω(container).ShouldNot(BeNil())
```

Comment:
OK, I'm going to request that we don't use `Ω`, but we use `Expect`, and to provide some semi-random other use case, it looks like the kubernetes end to end tests use ginkgo, and use `Expect`:
https://github.com/kubernetes/kubernetes/blob/master/test/e2e/common/init_container.go#L86

@sboeuf I suspect you just used the idiom that was already in the file - that's fine, but can you edit your commit to be Expect. I've opened #192 so we can track conversion of the rest of the `Ω` calls conversion.